overdrive.py

```diff
@@ -987,21 +992,22 @@ def to_collection(self, _db):
         collection, Overdrive Advantage collection)
         """
         # First find the parent Collection.
-        parent = get_one(
-            _db, Collection, external_account_id=self.parent_library_id,
-            protocol=Collection.OVERDRIVE
-        )
-        if not parent:
+        try:
+            parent = Collection.by_protocol(_db, ExternalIntegration.OVERDRIVE).filter(
+                Collection.external_account_id==self.parent_library_id
+            ).one()
+        except NoResultFound, e:
```

Comment:
With this and the implementation of `Collection.protocol` I decided that a collection with a parent inherits `.protocol` from its parent, along with other features of its `ExternalIntegration`. It doesn't need to be stored separately.